public/static/docs/user-guide/external-dependencies.md

```diff
@@ -150,8 +150,8 @@ determine whether the source has changed and we need to download the file again.
 
 ## Example: Using import
 
-`dvc import` can download a <abbr>data artifact</abbr> from any <abbr>DVC
-repository</abbr>. It also creates an external dependency in its <abbr>import
+`dvc import` can download a <abbr>data artifact</abbr> from any Git/DVC
+Repository. It also creates an external dependency in its <abbr>import
```

Comment:
> Repeat of https://github.com/iterative/dvc.org/pull/912#discussion_r366074082

No need to capitalize "repository" here. Actually, please use

`...from any <abbr>DVC project</abbr> or Git repository. It also...`

Thanks

Reply:
:+1: